processor/k8sprocessor/observability/observability.go

```diff
@@ -30,13 +30,15 @@ func init() {
 		viewPodsAdded,
 		viewPodsDeleted,
 		viewIPLookupMiss,
+		viewPodTableSize,
 	)
 }
 
 var (
-	mPodsUpdated = stats.Int64("otelsvc/k8s/pod_updated", "Number of pod update events received", "1")
-	mPodsAdded   = stats.Int64("otelsvc/k8s/pod_added", "Number of pod add events received", "1")
-	mPodsDeleted = stats.Int64("otelsvc/k8s/pod_deleted", "Number of pod delete events received", "1")
+	mPodsUpdated  = stats.Int64("otelsvc/k8s/pod_updated", "Number of pod update events received", "1")
+	mPodsAdded    = stats.Int64("otelsvc/k8s/pod_added", "Number of pod add events received", "1")
+	mPodsDeleted  = stats.Int64("otelsvc/k8s/pod_deleted", "Number of pod delete events received", "1")
+	mPodTableSize = stats.Int64("otelsvc/k8s/pod_table_size", "Size of table containing pod info", "1")
```

Comment:
Looks good, but is there a list of those metrics somewhere, telling users of this processor what they actually mean? Also, it would be a good opportunity to:

- s/otelsvc/otelcol
- replace the slashes by underscores, as it seems to be the standard at other metrics

If you don't want to do them here, feel free to open a tracking issue.

Reply:
@jpkrohling I've created an issue and also I'll start work on it https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2322